src/renderer/corporate/member/corporate-member-container.jsx

```diff
@@ -153,15 +153,19 @@ class CorporateMemberContainerComponent extends PureComponent {
 			case 'parentId':
 				return this.validateAttributeParentId(value);
 			case 'positions':
-				return true;
+				return value.length > 0;
 		}
 
-		const type = corporateAttributeTypes[name] || individualAttributeTypes[name];
+		const type =
+			this.state.type === 'individual'
+				? individualAttributeTypes[name]
+				: corporateAttributeTypes[name];
 
 		if (!type || !type.content) {
 			throw new Error(`${name} is not a basic attribute`);
 		}
 
+		console.log(type);
```

Comment:
stray log

I usually prepend "XXX" to debug logs, that way I can do a quick search on "XXX" before submitting and removing them